iosApp/iosApp/Pages/TripDetails/TripDetailsStopView.swift

```diff
@@ -26,12 +26,36 @@ struct TripDetailsStopView: View {
                         Spacer()
                         UpcomingTripView(prediction: upcomingTripViewState, routeType: routeType)
                     }
+                    .accessibilityElement()
```

Comment:
suggestion: If `.accessibilityElement` is removed here, or used with [.combine](https://developer.apple.com/documentation/swiftui/accessibilitychildbehavior/combine), then I think this reads as-intended without needing to specify a separate `accessibilityLabel` since the inner components of the view already have accessibility labels.

The connection piece of the accessibilityLabel could be added directly to `scrollRoutes`.

Reply:
I had that approach initially but [this post ](https://www.hackingwithswift.com/books/ios-swiftui/hiding-and-grouping-accessibility-data) told me that the `.combine` option will cause the text to be read with pauses between elements, rather than as a single statement. I figured using a separate label would be a smoother experience.

Reply:
Ah got it, thanks for that resource! I'm not sure if that is necessary in this case since there is already punctuation separating each constituent element in this custom accessibility Label - comma after stop name, period after arriving at. 

However, if this does all need to be one label, I think it would be worth using the logic from `UpcomingTripAccessibilityFormatters` for the time piece since that encompasses more scenarios than "arriving at"